src/test/java/FlattenerTest.java

```diff
@@ -0,0 +1,24 @@
+import com.scrive.pdftools.tools.flattener.FlattenerSpec;
+import com.scrive.pdftools.tools.flattener.LambdaFlattener;
+import org.json.JSONObject;
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.io.*;
+
+public class FlattenerTest {
+    @Test
+    public void shouldFlattenFields() throws IOException {
+        final InputStream fileToFlatten = new FileInputStream(TestUtils.inputFile("flat_test_filled.json"));
+        final JSONObject specToFlatten = new JSONObject(TestUtils.fromStream(fileToFlatten));
```

Comment:
probably can be replaced by `new JSONObject(IOUtils.resourceToString("flat_test_filled.json"))`

Reply:
Yes. Done

Reply:
Yes. Replaced